torchvision/prototype/transforms/_augment.py

```diff
@@ -99,8 +96,11 @@ def _transform(self, input: Any, params: Dict[str, Any]) -> Any:
             return input
 
     def forward(self, *inputs: Any) -> Any:
-        if torch.rand(1) >= self.p:
-            return inputs if len(inputs) > 1 else inputs[0]
+        sample = inputs if len(inputs) > 1 else inputs[0]
+        if has_any(sample, features.BoundingBox, features.SegmentationMask):
+            raise TypeError(f"BoundingBox'es and SegmentationMask's are not supported by {type(self).__name__}()")
+        elif torch.rand(1) >= self.p:
+            return sample
```

Comment:
Nit: (Can't comment on the right spot.) 2 lines below you pass `super().forward(*inputs)` but in other methods you do `return super().forward(sample)`. Choose one idiom.